packages/components/tests/integration/components/hds/form/textarea/base-test.js

```diff
@@ -24,6 +24,17 @@ module('Integration | Component | hds/form/textarea/base', function (hooks) {
     assert.dom('#test-form-textarea').hasValue('abc123');
   });
 
+  // ROWS
+
+  test('it should render the textarea with the default number of rows', async function (assert) {
+    await render(hbs`<Hds::Form::Textarea::Base />`);
+    assert.dom('textarea[rows="4"]').exists();
+  });
+  test('it should render the textarea with a custom number of rows', async function (assert) {
+    await render(hbs`<Hds::Form::Textarea::Base rows="2" />`);
+    assert.dom('textarea[rows="2"]').exists();
```

Comment:
maybe I would use something more explicit about what we're testing
```suggestion
    assert.dom('textarea').hasAttribute('rows', '2');
```